cilium/ebpf.io-summit
# feat: add talk title to Speakers & Grid Schedule in Topics

**Describe what changes this pull request brings**
- For Hero:
- [x] change title 'Summit 2021' to 'eBPF Summit 2021'
- For Featured Speakers:
- [x] add talk titles
- [x] add Sessionize page for each speaker
- For Topics:
- [x] replace the Topics section with an embedded Schedule Grid from Sessionize
- For Agenda:
- [x] replace link with a button


**Describe any additional changes**


**Steps to test**
1. Visit [the page](https://deploy-preview-2--ebpf-summit-2021.netlify.app/summit-2021/)
2. Confirm that everything looks and works as expected


**Screenshots**
Featured Speakers:
![image](https://user-images.githubusercontent.com/48465000/127828431-dad73a7a-1adb-46ec-8366-2b816bb948fb.png)
Agenda:
![image](https://user-images.githubusercontent.com/48465000/127828486-e186b4e7-7d72-4b3c-9e41-5dbd1cf3ffd4.png)
Schedule:
![image](https://user-images.githubusercontent.com/48465000/127829462-3f8e1a1a-49b3-4044-b49f-06ab1aa3e568.png)



**Checklist**
